### ISSUE TITLE
docker installation problem - failed to load symfony/flex (v1.0.80) from cache ...

### ISSUE THREAD

**Issue by AbsSimon** [api-platform/api-platform#793]
I just try to install my first distribution with docker ce (Windows) and get this error message.

```
    54/58:      https://codeload.github.com/doctrine/doctrine2/legacy.zip/87ee409783a4a322b5597ebaae558661404055a7
    55/58:      https://codeload.github.com/symfony/http-foundation/legacy.zip/4f9c7cf962e635b0b26b14500ac046e07dbef7f3
    56/58:      https://codeload.github.com/symfony/validator/legacy.zip/f2523bfd8dc5ff648aca55c0f2748674ca4661bb
    57/58:      https://codeload.github.com/twigphp/Twig/legacy.zip/7b604c89da162034bdf4bb66310f358d313dd16d
    58/58:      https://codeload.github.com/api-platform/core/legacy.zip/5b5b4ef13a70b495dda32863935a0a0f79873da1
    Finished: success: 58, skipped: 0, failure: 0, total: 58
Package operations: 58 installs, 0 updates, 0 removals
  - Installing symfony/flex (v1.0.80): Loading from cache
Plugin installation failed, rolling back
  - Removing symfony/flex (v1.0.80)


  [ErrorException]
  Declaration of Symfony\Flex\ParallelDownloader::getRemoteContents($originUrl, $fileUrl, $context) should be compatible with Composer\Util\RemoteFilesystem::getRemoteContents($originUrl, $fileUrl, $context, ?array &$responseHeaders = NULL)


install [--prefer-source] [--prefer-dist] [--dry-run] [--dev] [--no-dev] [--no-custom-installers] [--no-autoloader] [--no-scripts] [--no-progress] [--no-suggest] [-v|vv|vvv|--verbose] [-o|--optimize-autoloader] [-a|--classmap-authoritative] [--apcu-autoloader] [--ignore-platform-reqs] [--] [<packages>]...

ERROR: Service 'php' failed to build: The command '/bin/sh -c set -eux;         composer install --prefer-dist --no-dev --no-autoloader --no-scripts --no-progress --no-suggest;        composer clear-cache' returned a non-zero code: 1
```
does anyone know how to solve the problem? thank you & best regards

**Comment by guido4000** [comment]
I experience the same issue with Docker version 18.06.0-ce, build 0ffa825.

**Comment by mrkisch** [comment]
Same error.
On Windows 10
Docker:
Version 18.06.0-ce-win72 (19098)
Channel: stable
5513f91

**Comment by mrkisch** [comment]
#795 i might have find a solution

**Comment by guido4000** [comment]
> #795 i might have find a solution

Using symfony/flex v1.0.89 did not help in my case.

**Comment by sseidenthal** [comment]
i had the same issue, i did:
`composer update symfony/flex --no-scripts`

**Comment by guido4000** [comment]
> i had the same issue, i did:
`composer update symfony/flex --no-scripts`

Yes, adding that line to the dockerfile before the `composer install ..` command solves the issue.

**Comment by guido4000** [comment]
> Where should i add composer update symfony/flex --no-scripts in which file of api platform

In the api folder in the Dockerfile before line 72 add
`composer update symfony/flex --no-scripts; \`

**Comment by ragboyjr** [comment]
@sseidenthal thank you very much! Had the same issue, running from inside the api dir fixed the issue.

@guido4000 should a `composer update` command be run from inside of the Dockerfile? From my experience, `composer update` shouldn't be added into a docker file because it's not a stable command.

**Comment by guido4000** [comment]
> @guido4000 should a composer update command be run from inside of the Dockerfile? From my experience, composer update shouldn't be added into a docker file because it's not a stable command.

This was just a workaround. I suppose the dependencies will be updated in the official build soon.

**Comment by ragboyjr** [comment]
gotcha, no worries.

**Comment by xclaerho** [comment]
tried ijj

**Comment by teohhanhui** [comment]
Fixed in f3e5091
